include/dlaf/schedulers.h

```diff
@@ -50,6 +51,6 @@ auto getBackendScheduler(
 
 inline auto getMPIScheduler() {
   return pika::execution::experimental::thread_pool_scheduler{
-      &pika::resource::get_thread_pool(getConfiguration().mpi_pool)};
+      &pika::resource::get_thread_pool(pika::mpi::experimental::get_pool_name())};
 }
```

Comment:
This function can now be removed entirely (along with the `pika/mpi.hpp` include) since `transform_mpi` internally handles the transfer to/from the MPI pool.